wcivf/apps/elections/views/mixins.py

```diff
@@ -100,12 +96,8 @@ def people_for_ballot(self, postelection, compact=False):
         else:
             order_by = ["name_for_ordering", "person__name"]
 
-        people_for_post = people_for_post.order_by(
-            F("elected").desc(nulls_last=True), *order_by
-        )
-        people_for_post = people_for_post.order_by(
-            F("votes_cast").desc(nulls_last=True), *order_by
-        )
+        people_for_post = people_for_post.order_by("elected", "-votes_cast", *order_by)
```

Comment:
This makes sense. I think we need to keep the `F("elected").desc(nulls_last=True)` though, so this line should look like:

```
people_for_post = people_for_post.order_by(F("elected").desc(nulls_last=True), F("votes_cast").desc(nulls_last=True), *order_by)
```

(needs checking)